src/Runner.Worker/Handlers/ScriptHandler.cs

```diff
@@ -23,109 +23,113 @@ public sealed class ScriptHandler : Handler, IScriptHandler
 
         public override void PrintActionDetails(ActionRunStage stage)
         {
-            if (stage == ActionRunStage.Post)
+            // We don't want to display the internal workings if composite (similar/equivalent information can be found in debug)
```

Comment:
var writeDetails = ExecutionContext.InsideComposite ? ExecutionContext.Debug : ExecutionContext.Output as Action<string>;

...

writeDetails($"##[group]Run {firstLine}");

Reply:
I tried using delegate stuff but couldn't figure it out. 

I'm just going to do this: 
```csharp
void writeDetails(string message)
            {
                if (ExecutionContext.InsideComposite)
                {
                    ExecutionContext.Debug(message);
                }
                else 
                {
                    ExecutionContext.Output(message);
                }
            }
```